src/alacris_lib/side_chain.ml

```diff
@@ -302,9 +301,17 @@ module State = struct
     type nonrec t = t
     let marshaling = marshaling_of_rlping rlping
     let walk_dependencies _methods context {accounts; transactions; main_chain_transactions_posted} =
+      Logging.log "side_chain, walk_dependencies, step 1 digest(accounts)=%s" (Digest.to_string (AccountMap.digest accounts));
       walk_dependency AccountMap.dependency_walking context accounts
-      >>= fun () -> walk_dependency TransactionMap.dependency_walking context transactions
-      >>= fun () -> walk_dependency DigestSet.dependency_walking context main_chain_transactions_posted
+      >>= fun () ->
+      Logging.log "side_chain, walk_dependencies, step 2 digest(transactions)=%s" (Digest.to_string (TransactionMap.digest transactions));
```

Comment:
I systematically added statement of the kind
let ethereum_user_log = false/true (default is false)
So, now by default no-log but this can be implemented as required.